charts/greptimedb/values.yaml

```diff
@@ -25,8 +29,15 @@ datanode:
 greptimedb-etcd:
   etcdImage: greptime/etcd:v3.5.5
 
+initializerImage:
```

Comment:
- initializerImage -> initializer woulbe be better; 
- no need to use another `registry`, use `image.registry` instead;